Makefile

```diff
@@ -507,12 +502,10 @@ test-e2e: test-e2e-setup install-ginkgo
 	ginkgo run -mod=mod tests/e2e/ -- \
 	-settings=$(SETTINGS_TMP)/oadpcreds \
 	-provider=$(CLUSTER_TYPE) \
-	-creds_secret_ref=$(CREDS_SECRET_REF) \
 	-credentials=$(OADP_CRED_FILE) \
 	-ci_cred_file=$(CI_CRED_FILE) \
 	-velero_namespace=$(OADP_TEST_NAMESPACE) \
 	-velero_instance_name=$(VELERO_INSTANCE_NAME) \
-	-timeout_multiplier=$(E2E_TIMEOUT_MULTIPLIER) \
```

Comment:
@weshayutin this was your feature. It's going away in this PR. WDYT

Reply:
I removed because I think we never used

this would be helpful if there is a cluster scenario slower than CI itself

Reply:
There was indeed a cluster scenario than CI itself. Or rather.. CI is timing out.. so we were debugging by increasing this in local tests see if that solves it.

Reply:
Resolving per office hours